.github/workflows/webvoyager-benchmark.yml

```
@@ -0,0 +1,174 @@
+          description: "Custom user agent (optional)"
+          required: false
+          type: string
+          default: ""
+        task_dir:
+          description: "Path relative to notte_eval/data with the tasks json"
+          required: false
+          type: string
+          default: "webvoyager/webvoyager_simple.jsonl"
+
+concurrency:
+  group: >-
```

actionlint (1.7.7)

property "task_set_name" is not defined in object type {headless: string; max_steps: string; model: string; n_runs: string; proxies: string; task_dir: string; use_sdk: string; use_vision: string; user_agent: string}

(expression)